xblockutils/resources.py

```diff
@@ -51,8 +52,24 @@ def render_django_template(self, template_path, context=None):
         Evaluate a django template by resource path, applying the provided context
         """
         context = context or {}
+        libraries = {
+            'i18n': 'xblockutils.templatetags.i18n',
+        }
+
+        # If django>1.8, Engine can load the extra templatetag libraries
+        if django.VERSION[0] > 0 and django.VERSION[1] > 8:
+            engine = Engine(libraries=libraries)
+        else:
+            # For django 1.8, we have to load the libraries manually
+            from django.template import base as TemplateBase
+            for library_name in libraries:
+                library = TemplateBase.import_library(libraries[library_name])
+                if library:
+                    TemplateBase.libraries[library_name] = library
```

Comment:
This is changing global state in `TemplateBase` module. Would it make sense to we revert the changes made to `TemplateBase.libraries` after we are done rendering the template, so that these changes don't affect template rendering in unrelated parts of the codebase?